### PR TITLE
rambox: 0.5.13 -> 0.5.17

### PR DESCRIPTION
**Pull request by dtzWill**
Remove patch that is apparently already applied.

I've never used this app before,
but it seems to work as expected.

cc @gnidorah (maintainer)

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by dtzWill** [comment]
Bumped sencha too while visiting this.

**Comment by dtzWill** [comment]
> Similar software Franz is now FOSS https://github.com/meetfranz/franz No non-free components like sencha are used to build it. We could package it once it will get first release.

Excellent!

Actually we already seem to have a version (4.0.4) packaged!

**Comment by gnidorah** [comment]
@dtzWill yep, though 4.x branch is proprietary ;-)

**Comment by dtzWill** [comment]
Ah, okay.  Welp, looking forward to the release! Thanks for pointing to it!

File: pkgs/applications/networking/instant-messengers/rambox/hide-check-for-updates.patch [hunk 1]
```diff
@@ -1,35 +0,0 @@
-https://github.com/saenzramiro/rambox/issues/1283
```
Comment (yegortimoshenko) [inline pkgs/applications/networking/instant-messengers/rambox/hide-check-for-updates.patch]: Now that this patch is gone, we should patch `--without-update` option: https://github.com/saenzramiro/rambox/issues/1283

**Comment by dtzWill** [comment]
Not sure I follow, sorry.

**Comment by yegortimoshenko** [comment]
This patch removed update notifications and "check for updates" functionality. Upstream now exposes `--without-update` flag that can be used to the same effect. We should wrap `rambox` executable via `wrapProgram` so that we always pass this flag.

**Comment by dtzWill** [comment]
Okay, got it thanks.  Happily it appears wee already do this? https://github.com/dtzWill/nixpkgs/blob/9bc8b64894229f0a2005156cbc30545fee7e19cf/pkgs/applications/networking/instant-messengers/rambox/default.nix#L36

**Comment by yegortimoshenko** [comment]
Oh, I forgot about that, sorry. Looks good to me otherwise!

**Comment by gnidorah** [comment]
trying https://cdn.sencha.com/cmd/6.5.3/no-jre/SenchaCmd-6.5.3-linux-amd64.sh.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
curl: (22) The requested URL returned error: 404 Not Found
error: cannot download SenchaCmd-6.5.3-linux-amd64.sh.zip from any mirror
builder for '/nix/store/8f44dypiy9jklmvn9d0v0lfcgdkcclk8-SenchaCmd-6.5.3-linux-amd64.sh.zip.drv' failed with exit code 1

**Comment by dtzWill** [comment]
Ruhroh! Well I can confirm that link is dead, lame that it's so short-lived. New url (from https://www.sencha.com/products/extjs/cmd-download/ under Linux 64bit no-jre)) appears to be:

http://cdn.sencha.com/cmd/6.5.3.6/no-jre/SenchaCmd-6.5.3.6-linux-amd64.sh.zip

And of course sha256 will need to be fixed. And at least basic testing done to ensure everything still works but TBH I would very much expect it will.

I am AFK (sending from phone) and won't be able to put together a PR for this until tomorrow at the earliest (sorry!).  Hopefully this is enough information to workaround the problem or for someone else to submit a PR with the fix. Hope this helps, bug me if this isn't fixed soon. Thanks!

**Comment by gnidorah** [comment]
Done https://github.com/NixOS/nixpkgs/pull/42577
@dtzWill @yegortimoshenko Please merge my friend first PR (trivial two-line change) https://github.com/NixOS/nixpkgs/pull/42516

**Pull request merged**